e2e/assertions.js

```diff
@@ -14,13 +15,11 @@ export const assertMintedTbtcAmount = (web3, deposit, expectedTbtcAmount) => {
 }
 
 export const assertTbtcAccountBalance = async (
```

Comment:
Still used in `e2e-test.js` in the old way (lines 128 and 153)

Reply:
These guys were already removed in the [prev](https://github.com/keep-network/local-setup/pull/39/commits/fb36b5bcdc54971cb1a126c523f4e8941ab8814a) commit. Do you have the latest?

Reply:
@lukasz-zimnoch I've fixed the input parameters for `assertTbtcAccountBalance()` in `12cb3b1` commit